pkg/collector/corechecks/containers/kubelet/testdata/pods_requests_limits.json

```diff
@@ -119,7 +119,7 @@
               },
               "requests":{
                 "cpu":"500m",
-                "ephemeral-storage":"500m",
+                "ephemeral-storage":"500Mi",
```

Comment:
Note for reviewers: I changed this to make it more realistic, as `500m` for memory means 0.5 bytes.